portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -87,4 +97,21 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
   private Optional<String> getParameter(HttpServletRequest request, String name) {
     return Optional.ofNullable(request.getParameter(name));
   }
+
+  /** Returns the choice entered by the player, or -1 if the choice was invalid. */
```

Comment:
nit: choice entered by played? Change the wording here.

Reply:
Oops! Thanks.